packages/web/src/instrumentations/console/index.ts

```diff
@@ -5,7 +5,7 @@ export interface ConsoleInstrumentationOptions {
 }
 
 export class ConsoleInstrumentation extends BaseInstrumentation {
-  static defaultDisabledLevels: LogLevel[] = [LogLevel.DEBUG, LogLevel.LOG];
+  static defaultDisabledLevels: LogLevel[] = [LogLevel.DEBUG, LogLevel.TRACE];
```

Comment:
Also, don't forget to update the README file for this.

Reply:
- [ ] update readme